packages/zoltan2/CMakeLists.txt

```diff
@@ -48,9 +48,12 @@ ENDFOREACH()
 #
 # B) Set up package-specific options
 #
-
 IF (${PROJECT_NAME}_ENABLE_Epetra)
- GLOBAL_SET(HAVE_${PACKAGE_NAME_UC}_EPETRA ON)
+  IF (DEFINED Zoltan2_ENABLE_Epetra)
```

Comment:
Epetra is an optional library dependency of Zoltan2, so `Zoltan2_ENABLE_Epetra` should always be defined.